compositor_pipeline/src/audio_mixer.rs

```diff
@@ -68,18 +67,29 @@ impl AudioMixer {
         self.0.lock().unwrap().update_output(output_id, audio)
     }
 }
+const VOL_DOWN_THRESHOLD: f64 = i16::MAX as f64;
+const VOL_UP_THRESHOLD: f64 = 0.7f64 * i16::MAX as f64;
```

Comment:
Please don't use f64 suffix in the literal definition. Initially, I was fine with that, but it is really hard to quickly parse those values when quicklly looking through code